pkg/ctrlz/ctrlz_test.go

```diff
@@ -16,8 +16,13 @@ package ctrlz
 
 import "testing"
 
-func TestStartStop(t *testing.T) {
+func TestRunClose(t *testing.T) {
 	o := DefaultOptions()
-	go Run(o, nil)
-	Stop()
+	s := Run(o, nil)
+	s.Close()
```

Comment:
This is still racy and `httpServer.Close` can be invoked before `httpServer.ListenAndServe`. Creating a listener directly first (and returning error if it fails) before invoking `httpServer.Serve` avoids this problem (see https://github.com/istio/istio/pull/7613/commits/344c90e2224f8d1e6bc99d6a48b8739a0d83de80#diff-bb9e662310b0de50b15e76e87d187e18R183).